Classes/Command/DbDocs/Generator.php

```diff
@@ -87,7 +83,7 @@ public function injectConfigurationManager(ConfigurationManager $configurationMa
 
     public function __construct()
     {
-
+        $this->languageService = GeneralUtility::makeInstance(LanguageService::class);
```

Comment:
@thomaslow, this line throws a runtime exception when I run `vendor/bin/typo3 kitodo:dbdocs`:
```
Exception has occurred.
ArgumentCountError: Too few arguments to function TYPO3\CMS\Core\Localization\LanguageService::__construct(), 0 passed in /typo3_12/vendor/typo3/cms-core/Classes/Utility/GeneralUtility.php on line 2985 and exactly 3 expected
```